src/sidebar/components/annotation-body.js

```diff
@@ -11,8 +11,24 @@ import MarkdownView from './markdown-view';
 import TagEditor from './tag-editor';
 import TagList from './tag-list';
 
+/** @typedef {import("../../types/api").Annotation} Annotation */
+
+/**
+ * @typedef AnnotationBodyProps
+ * @prop {Annotation} props.annotation - The annotation in question
```

Comment:
The `props.` prefixes are not needed here since you're defining a type explicitly rather than creating an "inline" type.

Reply:
oops, missed that!